services/authutil/src/main/java/org/sagebionetworks/authutil/SimpleCORSFilter.java

```diff
@@ -31,7 +30,9 @@
 	public static final String ACCESS_CONTROL_REQUEST_METHOD = "Access-Control-Request-Method";
 	public static final String ALL_ORIGINS = "*";
 	public static final String METHODS = "POST, GET, PUT, DELETE";
-	public static final String HEADERS = "Origin, X-Requested-With, Content-Type, Accept, " + AuthorizationConstants.SESSION_TOKEN_PARAM;
+	public static final String HEADERS = "Origin, X-Requested-With, Content-Type, Accept, " + 
+			AuthorizationConstants.SESSION_TOKEN_PARAM +
+		", "+AuthorizationConstants.AUTHORIZATION_HEADER_NAME;

```

Comment:
Double check that if the Auth header is considered a "custom" header?

Reply:
Per Jay, this is correct.